core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -222,6 +226,52 @@ protected function getDateString()
 				. 'week:"' . $GLOBALS['TL_LANG']['DP']['week'] . '"'
 			. '});';
 	}
+
+	/**
+	 * Add the contao.backend configuration
+	 */
+	private function addBackendConfig(): void
+	{
+		$container = System::getContainer();
+
+		if (!$container->hasParameter('contao.backend'))
+		{
+			return;
+		}
+
+		$backendConfig = $container->getParameter('contao.backend');
+
+		if (!empty($backendConfig['attributes']) && \is_array($backendConfig['attributes']))
+		{
+			$this->attributes = ' ' . implode(' ', array_map(
```

Comment:
@leofeyer  why did you revert to my first solution? I discussed that with @m-vo and I refactored to loop in templates to have access to all raw attributes in the template!

Reply:
Why do you need access to the raw attributes in the template? We are adding custom attributes as `<?= $this->attributes ?>` everywhere else, too.

Reply:
Yes for me it's also ok - just if someone would do any customization depending on the attributes ;-)
The current solution was also my first attempt 👍

Reply:
I really dislike having a prerendered string. This makes it unusable to ship anything custom to the templates. If it's an associative array I can ouput the results at several places and can decide if I want to use attributes or anything else.

We had the same discussion with the link attributes and options of the `Figure` object btw. Imo this is the template's job - and If it's cumbersome to implement, we should add a template method.

Reply:
So we're adding a template method under the same name, so that it's BC compatible? :fox_face:

Reply:
Yes, something like this:

```php
class Template
{
    public function attributes(array $attributes)
    {
        return ' ' . implode(' ', array_map(
            static function ($v, $k) { return sprintf('%s="%s"', $k, $v); },
            $attributes,
            array_keys($attributes)
        ));
    }
}
```

Reply:
Or (then it's BC compatible without the need of a second variable):
```php
class TemplateAttributes extends ArrayIterator {
  public function __toString(): string {
        return ' ' . implode(' ', array_map(
            static function ($v, $k) { return sprintf('%s="%s"', $k, $v); },
            $this->array,
            array_keys($this->array)
        ));
  }
}
```
```php
// use
$this->Template->attributes = new TemplateAttributes($attributes);

// consume
echo $this->attributes; 

foreach($this->attributes as $attribute) {
  // ...
}
```

Reply:
I like it, however, people are likely to have `$this->Template->attributes = ' data-custom="123"` in their custom extensions, which would overwrite the `TemplateAttributes` object with a string again.

Reply:
Sounds good for me so for now we keep it as it is in this PR?

Reply:
Yes, that is what I would prefer.